### ISSUE TITLE
not working with bootstrap dialog

### ISSUE THREAD

**Issue by calidion** [eternicode/bootstrap-datepicker#1251]
latest version downloaded with bower

**Comment by acrobat** [comment]
Can you provide a jsfiddle example displaying the problem? (you can use [this base jsfiddle](http://jsfiddle.net/g02L8z8y/))

**Comment by calidion** [comment]
you can simply create a modal dialog with a form  and add an input with datepicker classname. you will see if the date selector show or not.

**Comment by hebbet** [comment]
i can't reproduce that in multiple of my projects. A simple jsfiddle that shows the bug, can help us.

**Comment by Tolerant** [comment]
Hi, same problem with dialog.
http://jsfiddle.net/7mder0oa/
input dublicate on click...
-
solution, rename event show.bs.modal to shown.bs.modal

**Comment by acrobat** [comment]
Yes the collision of the bsModal events and ours is indeed a problem. [There are a few issues and pr's](https://github.com/eternicode/bootstrap-datepicker/labels/namespaced%20events) to fix this problem but i'm going to look in to a way to fix this as it causes a BC break

**Comment by calidion** [comment]
it seems caused by the z-index value, but i cannot found the dialog, and i am using jquery dialog instead.

**Comment by Tolerant** [comment]
acrobat, this fix wont work for me.

**Comment by Eric75015** [comment]
do you find a solution for the submit button? still not working for me, I click on submit and datepicker popup! I'm not in modal...thanks

**Comment by nitusharma1982** [comment]
How this problem has been solved?
I am facing similar issue

**Comment by Azaret** [comment]
As `1.7.0` is seems to work well.
Could you download the lastest, test and confirm please ?

Just for testing purpose, don't push it in production, `1.7.0` is in development stage.
